roles/aap_setup_install/defaults/main.yml

```
@@ -66,4 +66,7 @@ aap_setup_inst_force: false
 # review before bumping up the default AAP version
 aap_setup_inst_fixes:
   - aap_4555  # setup.log is only saved on first call as non-root (2.2.0)
+
+# perform postinstallation validation
+aap_postinstall_validation: true
```

The prefix is the (shortened) name of the role, by good practice. Please adhere to it.
```suggestion
aap_setup_inst_validation: true
```